pkg/scheduler/internal/cache/cache.go

```diff
@@ -518,10 +537,9 @@ func (cache *schedulerCache) UpdatePod(oldPod, newPod *v1.Pod) error {
 			klog.ErrorS(nil, "SchedulerCache is corrupted and can badly affect scheduling decisions")
 			os.Exit(1)
 		}
-		if err := cache.updatePod(oldPod, newPod); err != nil {
+		if err := cache.updatePod(oldPod, newPod, key); err != nil {
```

Comment:
Can we eliminate the `key` parameter as it can be obtained from the pod?

Reply:
Sure, of course